pkgs/tools/security/tessen/default.nix

```
@@ -0,0 +1,46 @@
+{ lib
+, stdenvNoCC
+, fetchFromSourcehut
+, pass
+, bemenu
+, passBackend ? pass
+, picker ? bemenu
+, wtype
+, wl-clipboard
+}:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "tessen";
+  version = "2.1.2";
+
+  src = fetchFromSourcehut {
+    owner = "~ayushnix";
+    repo  = pname;
+    rev = "8758a09345f6eef24764de4a0efad737f12562c8";
```

```suggestion
    rev = "v${version}";
```
I hope this also works for ffsh as it does for ffgh.